auction.py

```diff
@@ -209,6 +213,7 @@ async def nominate(self, ctx):
 
       # We have a nomination, run the lot
       print(f"Starting lot {self.auction.current_lot.to_dict()}")
+      await ctx.send(embed = embed.player_info(ctx.message))
```

Comment:
not pep8. `await ctx,send(embed=embed...)`

Reply:
Fixed